packages/EasyTest/src/InvalidDataMaker/InvalidDataMaker.php

```diff
@@ -0,0 +1,437 @@
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldArrayCollectionWithMoreItems(int $maxElements): iterable
+    {
+        $value = new ArrayCollection(\array_fill(0, $maxElements - 1, null));
+        $message = $this->translateMessage(
+            (new Count([
+                'max' => $maxElements,
+            ]))->maxMessage,
+            [
+                '{{ limit }}' => $maxElements,
+            ],
+            $maxElements
+        );
+
+        yield from $this->create("{$this->property} has too more elements in the collection", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldArrayWithFewerItems(int $minElements): iterable
+    {
+        $value = \array_fill(0, $minElements - 1, null);
+        $message = $this->translateMessage(
+            (new Count([
+                'min' => $minElements,
+            ]))->minMessage,
+            [
+                '{{ limit }}' => $minElements,
+            ],
+            $minElements
+        );
+
+        yield from $this->create("{$this->property} has too few elements in the array", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldArrayWithMoreItems(int $maxElements): iterable
+    {
+        $value = \array_fill(0, $maxElements + 1, null);
+        $message = $this->translateMessage(
+            (new Count([
+                'max' => $maxElements,
+            ]))->maxMessage,
+            [
+                '{{ limit }}' => $maxElements,
+            ],
+            $maxElements
+        );
+
+        yield from $this->create("{$this->property} has too many elements in the collection", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldBlankString(): iterable
+    {
+        $value = '';
+        $message = $this->translateMessage((new NotBlank())->message);
+
+        yield from $this->create("{$this->property} is blank", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldDateTimeLessThanOrEqualToNow(): iterable
+    {
+        $dateTime = Carbon::now();
+        $message = $this->translateMessage(
+            (new GreaterThan([
+                'value' => 'now',
+            ]))->message,
+            [
+                '{{ compared_value }}' => 'now',
+            ]
+        );
+
+        $value = $dateTime->clone()
+            ->subSecond()
+            ->toAtomString();
+        yield from $this->create("{$this->property} has less datetime", $value, $message);
+
+        $value = $dateTime->toAtomString();
+        yield from $this->create("{$this->property} has equal datetime", $value, $message);
+    }
+
+    /**
+     * @return iterable<mixed>
+     */
+    public function yieldEmptyArrayCollection(int $minElements): iterable
```

Comment:
Why don't you simply reuse the `yieldArrayCollectionWithFewerItems` method here?

Reply:
If we use `yieldArrayCollectionWithFewerItems` here then will be available only empty collection with `minElements = 1`. I think it will be usefull to have different  `minElements`.